src/terrain/routes.clj

```diff
@@ -75,8 +75,10 @@
    the next set of routes (`secured-routes-no-context`) if nothing matches."
   []
   (util/flagged-routes
+   (apps-routes)
    (dashboard-aggregator-routes)
    (filesystem-stat-routes)
+   (quicklaunch-routes)
```

Comment:
Should these routes now be removed from `secured-routes-no-context` below?

Reply:
Ah, I didn't notice that section... I guess I don't know what it means to remove it from that section.  I tried removing them locally and nothing seemed to error out, endpoints still seem to work, so I'm not immediately seeing an issue with removing them from `secured-routes-no-context`.

Reply:
So all the routes have to be defined in `swagger-routes` at the bottom of this file, which is just using the route wrapper `terrain-routes`.

If you check out `terrain-routes`, you'll see the order the routes are actually defined in.
Since `optionally-authenticated-routes` are listed before `secured-routes-no-context`, then compojure-api will find these routes included in `optionally-authenticated-routes` first.

So it's not an error to also include the duplicate routes in `secured-routes-no-context`, but I think it's cruft that can be removed.

Reply:
Cool - thanks for the explanation!  I'm guessing it's only really a duplicate if the route originally appeared in `secured-routes-no-context` and not `secured-routes`?  Otherwise, I have more to dupes clean up 😄

Reply:
Ah, yes, good catch. Adding the routes to `secured-routes` does add the extra `/secured` context, so that does make a difference and the duplicates could be left in there.